README.md

```diff
@@ -18,13 +18,13 @@ This was originally created as a school project, intended to run on school serve
     ```env
     DB_HOST=marie
     # DB_HOST='127.0.0.1'
-    DB_PORT=3306
-    DB_USER=csci311h
-    DB_PASS=rAnhgtHT
-    DB_NAME=csci311h_tekku
-    PHRASE=tekku
+    DB_PORT=removed

```

Comment:
Hmm this is actually meant for if someone wanted to run the server and database locally, so I think we can probably keep the previous changes. The port, for example, is important, and shouldn't be set to "removed". But maybe the password should be set to something else, such as "change_me".